### ISSUE TITLE
Cache gone and unable to restore after kernel update in Arch Linux.

### ISSUE THREAD

**Issue by f3bruary** [stec-inc/EnhanceIO#98]
I updated my Arch Linux and it pulled a new kernel. After I rebooted I noticed some enhanceIO errors and it appears that the setup got nuked.

I re-installed enhanceio-dkms-git from AUR. Followed the Wiki: https://wiki.archlinux.org/index.php/EnhanceIO again. (files in /etc/initcpio/hook, /etc/initcpio/install were gone). Fixed permission errors and re-created the initcpio.

It still won't work.

>[f3bruary:~]$ eio_cli info
>No caches Found
>[f3bruary:~]$ sudo eio_cli create -d /dev/sda -s /dev/sdb -c my_first_enhanceio
>Cache Name       : my_first_enhanceio
>Source Device    : /dev/sda
>SSD Device       : /dev/sdb
>Policy           : lru
>Mode             : Write Through
>Block Size       : 4096
>Associativity    : 256
>Cache creation failed (dmesg can provide you more info)
>[f3bruary:~]$ dmesg | tail -1
>[ 3021.746817] enhanceio: Cache creation failed: Cache already exists.
>[f3bruary:~]$ sudo eio_cli delete -c my_first_enhanceio
>No cache exists with the name my_first_enhanceio

So I can't create a new cache and can't delete the old one...

**Comment by f3bruary** [comment]
I removed all files relating to enhanceio, uninstalled the app, removed files from the hooks and install folder and mkinicpio.conf, then regenerated it, rebooted and it seemed to be gone. Then I installed enhanceio again, followed the wiki, and 'eio_cli info' showed it was working. Then after a reboot it's the same issue again.
I notice in journactl it's executing each enhanceio command twice: http://pastebin.com/dzQXa2st

**Comment by f3bruary** [comment]
Is EnhanceIO deprecated ?

**Comment by flrichar** [comment]
Have you tried the command 'eio_cli enable' instead of create, if the cache is already existing?

**Comment by f3bruary** [comment]
No, but right now it's broken on the latest kernel (on Arch Linux) anyways. :(
